PostHog/posthog
# fix(property-select): trim whitespace from select options in PropertyFilters

Fixes #17254 - Cohorts property ID filter copy and pasting values bug

## Problem
If you try to create a cohort based on multiple values for filtering criteria e.g. organization_id, and then paste the list of values into the property filter, the values appear to paste properly in the field, but the cohort persons return based on the first ID only.

This is because of how the multi-select component handled the tokenization of the values. It expected a "comma" separated list in the strict sense so token1, token2, token3... was converted to "token1", " token2", "token3" etc.

## Changes
Trimmed the tokenized property filters after pasting.

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?
Manually:
- Paste a comma-separated list of values with spaces into the cohort filter
- Confirm that the cohort matches all of the specified values
